lib/index.js

```diff
@@ -3,23 +3,39 @@
  */
 'use strict';
 
-const {documentLoader} = require('bedrock-jsonld-document-loader');
-const jsigs = require('jsonld-signatures');
-const {Ed25519Signature2018} = jsigs.suites;
+const {jsonLdDocumentLoader} = require('bedrock-jsonld-document-loader');
+const {Ed25519Signature2020} = require('@digitalbazaar/ed25519-signature-2020');
 const {validateOperation, runOperation} = require('webkms-switch');
 const keystores = require('./keystores.js');
 const BedrockKmsModuleManager = require('./BedrockKmsModuleManager.js');
 const MongoDBKeyDescriptionStorage =
   require('./MongoDBKeyDescriptionStorage.js');
+
+const ed25519sig2018 = require('ed25519-signature-2018-context');
+const ed25519sig2020 = require('ed25519-signature-2020-context');
+const x25519ka2020 = require('x25519-key-agreement-2020-context');
+
 require('bedrock-did-context');
 require('bedrock-veres-one-context');
 
-const didIo = require('did-io');
+jsonLdDocumentLoader.addStatic(ed25519sig2018.constants.CONTEXT_URL,
+  ed25519sig2018.contexts.get(ed25519sig2018.constants.CONTEXT_URL));
+
+jsonLdDocumentLoader.addStatic(ed25519sig2020.constants.CONTEXT_URL,
+  ed25519sig2020.contexts.get(ed25519sig2020.constants.CONTEXT_URL));
+
+jsonLdDocumentLoader.addStatic(x25519ka2020.constants.CONTEXT_URL,
+  x25519ka2020.contexts.get(x25519ka2020.constants.CONTEXT_URL));
+
+const {CachedResolver} = require('@digitalbazaar/did-io');
+const resolver = new CachedResolver();
 
 // Config did-io to support did:key and did:v1 drivers
-didIo.use('key', require('did-method-key').driver());
+
+resolver.use(require('@digitalbazaar/did-method-key').driver());
+
 // TODO: Move v1 'test' mode param to config.
-didIo.use('v1', require('did-veres-one').driver({mode: 'test'}));
+resolver.use(require('did-veres-one').driver({mode: 'test'}));
```

Comment:
@JSAssassin we have a new module for this: https://github.com/digitalbazaar/bedrock-did-io/pull/2

Just grab the exported `didIo` here and use it where `didIo` was used here before.  The `didIo` export **is** a properly configured resolver.  No need for the `.use` API calls.

Reply:
Updated here c935241